example/fly_mission/fly_mission_dronecore.cpp

```diff
@@ -0,0 +1,357 @@
+//
+// Simple example to demonstrate how to use DroneCore.
```

Comment:
Maybe adapt this to say `Example to demonstrate how to fly misison using DroneCore`.